test/e2e/diff_test.go

```diff
@@ -165,4 +165,15 @@ RUN touch %s`, ALPINE, imagefile)
 		Expect(session.OutputToString()).To(ContainSubstring(confile))
 	})
 
+	It("podman diff without args", func() {
+		if !IsRemote() {
+			session := podmanTest.Podman([]string{"diff"})
+			session.WaitWithDefaultTimeout()
+			Expect(session).Should(ExitWithError(125, "diff requires a name or id"))
+		} else {
+			session := podmanTest.Podman([]string{"diff"})
+			session.WaitWithDefaultTimeout()
```

Comment:
There's no need to duplicate the command and wait. Just move this outside (before) the `if`, and leave only the different error messages in the conditional.